membership-attribute-service/app/models/Attributes.scala

```diff
@@ -15,13 +15,14 @@ object ContentAccess {
   implicit val jsWrite = Json.writes[ContentAccess]
 }
 
-case class Attributes(UserId: String, Tier: String, MembershipNumber: Option[String], PublicTier: Option[Boolean] = None) {
+case class Attributes(UserId: String, Tier: String, MembershipNumber: Option[String], PublicTier: Option[Boolean] = None, AdFree: Option[Boolean] = None) {
   require(Tier.nonEmpty)
   require(UserId.nonEmpty)
 
   lazy val allowsPublicTierDisplay = PublicTier.exists(identity)
   lazy val isFriendTier = Tier.equalsIgnoreCase("friend")
   lazy val isPaidTier = !isFriendTier
+  lazy val isAdFree = AdFree.exists(identity)
```

Comment:
The reference in the code is to a scala predef of an identity function that, apparently, always returns the same value as the passed argument. It's used to catch the `PublicTier` value, which is also an `Option[Boolean]`, so I figured it'd be ok for `AdFree` too 🤕